libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.ts

```diff
@@ -83,26 +84,15 @@ export class SkyHelpInlineComponent {
    * The label of the component the help inline button is attached to.
    */
   @Input()
-  public labelText: string | undefined;
+  public set labelText(value: string | undefined) {
+    this.#labelText.set(value);
```

Comment:
I'm setting this standard input value to a signal input, to avoid changing the public API. I'm open to a different approach.